CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+## nucleo-matcher 
+
+# [0.2.0] - 2023-09-01
+
+*initial public release*
+
+
+[0.2.0]: https://github.com/olivierlacan/keep-a-changelog/releases/tag/nucleo-v0.2.0
```

Comment:
Should this link point to the github release? Or will this link be published when the release is?

Reply:
A right I forgot to fully change the link from the template :D